docker-compose.yml

```diff
@@ -1,40 +1,36 @@
-version: '2'
```

Comment:
We probably want to move prod away from Compose anyway, now that the competition is over. That said, probably out of scope for this PR.